packages/bpk-component-accordion/README.md

```diff
@@ -77,6 +98,7 @@ export default () => (
 | id        | string   | true     | -             |
 | title     | string   | true     | -             |
 | expanded  | bool     | false    | false         |
+| icon      | node     | false    | null          |
```

Comment:
I wonder if this should be called something else, because you could use it to display anything (eg a badge).
In the past I think we've used "auxilliary view" 🤔

Reply:
Yeah - this was because we didn't want to open it up so much to consumers for abuse as it means people could put a button in there wish is undesirable

Reply:
I agree with Ollie, let's use the API name to implicitly limit it to icons.